torch/utils/data/dataframes_pipes.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.10"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3610jvsc74a57bd0eb5e09632d6ea1cbf3eb9da7e37b7cf581db5ed13074b21cc44e159dc62acdab",
+   "display_name": "Python 3.6.10 64-bit ('dataloader': conda)"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "## How DataFrames (DF) and DataPipes (DP) can work together"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torch.utils.data import IterDataPipe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Example IterDataPipe\n",
+    "class ExampleIterPipe(IterDataPipe):\n",
+    "    def __init__(self, range = 20):\n",
+    "        self.range = range\n",
+    "    def __iter__(self):\n",
+    "        for i in range(self.range):\n",
+    "            yield i"
+   ]
+  },
+  {
+   "source": [
+    "`to_dataframes_pipe` converts tuples into DataFrames, ideally dataloaders will load into DF directly"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "   i  j\n0  0  0\n1  1  1\n2  2  2\n3  3  0\n4  4  1\n5  5  2\n6  6  0\n    i  j\n0   7  1\n1   8  2\n2   9  0\n3  10  1\n4  11  2\n5  12  0\n6  13  1\n    i  j\n0  14  2\n1  15  0\n2  16  1\n3  17  2\n4  18  0\n5  19  1\n"
+     ]
+    }
+   ],
+   "source": [
+    "def get_dataframes_pipe(range = 20, batch_size = 7):\n",
+    "    return ExampleIterPipe(range = range).map(lambda i: (i, i % 3)).to_dataframes_pipe(columns = ['i','j'], batch_size = batch_size)\n",
+    "\n",
+    "dp = get_dataframes_pipe()\n",
+    "\n",
+    "# Iterator converts DF Pipe into singular items\n",
+    "for i in dp:\n",
+    "    print(i)"
```

Comment:
I have same feeling with Francisco that DFPipe should represent one DF and yield rows.